PositionBehavior.php

```diff
@@ -253,7 +253,7 @@ protected function createGroupConditionAttributes()
         $condition = [];
         if (!empty($this->groupAttributes)) {
             foreach ($this->groupAttributes as $attribute) {
-                $condition[$attribute] = $this->owner->$attribute;
+                $condition[$attribute] = empty($this->owner->$attribute)?null:$this->owner->$attribute;
```

Comment:
Fix is incorrect: empty value can still be valid for the group attribute. For example: `0` can be used.